draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1199,6 +1199,18 @@ Outputs:
   : A base64 encoded `TransItem` of type `x509_sct_v2` or `precert_sct_v2`,
     signed by this log, that corresponds to the `submission`.
 
+  If the submitted entry has already been appended to this log's tree, or if
```

Comment:
I'll reword that sentence as follows:
`If the submitted entry is immediately appended to, or already exists in, this log's tree, then the log SHOULD also output:`